h/models/user.py

```diff
@@ -214,7 +214,7 @@ def username(self, value):
         self._username = value
 
     @username.comparator
-    def username(cls):
+    def username(cls):  # noqa: N805
```

Comment:
I've checked that this is indeed a class method. See http://docs.sqlalchemy.org/en/latest/orm/extensions/hybrid.html and in particular the "Custom comparators" section.